feat(curriculum): add block 2 dialogs and tasks of the B1 course of the English Curriculum (#57471)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f76d4b7384c063c097048.md

---
id: 671f76d4b7384c063c097048
title: Task 96
challengeType: 22
dashedName: task-96
---

<!-- (Audio) Maria: While the speaker was going through neural networks, I thought about our project last year. -->

# --instructions--

Listen to the audio and complete the sentence below.

# --fillInTheBlank--

## --sentence--

`While the speaker was BLANK BLANK neural networks, I thought about our project last year.`

## --blanks--

`going`

### --feedback--

This verb is used as part of the phrasal verb `go through`, meaning to review or explain something in detail. Use it in the `-ing` form.

---

`through`

### --feedback--

This preposition is part of the phrasal verb `go through`, referring to a detailed explanation or review.

# --explanation--

> The phrasal verb `go through` means to review, explain, or examine something in detail. It is often used when covering material in a structured or methodical way. Example: `The instructor went through the project requirements step by step.` (this means the instructor checked the requirements in a detailed manner)  
> 
> `Neural networks` are a type of artificial intelligence inspired by the structure and function of the human brain. They consist of layers of nodes (neurons) that process data to solve problems, such as image recognition or natural language processing. Example: `The AI model uses neural networks to classify images with high accuracy.` (this means the AI model uses this structure based on the human brain to process data)
> 
> In this sentence, Maria describes how the speaker was explaining neural networks during the workshop, prompting her to think about a past project.
> 

Reviewer: ```suggestion
To `go through` means to review, explain, or examine something in detail. It is often used when covering material in a structured or methodical way. For example: 

`The instructor went through the project requirements step by step.` - This means the instructor checked the requirements in a detailed manner.  

`Neural networks` are a type of artificial intelligence inspired by the structure and function of the human brain. For example: 

`The AI model uses neural networks to classify images with high accuracy.` - This means the AI model uses this structure based on the human brain to process data.

In this sentence, Maria describes how the speaker was explaining neural networks during the workshop, prompting her to think about a past project.

```
